source/iterators.tex

```diff
@@ -1312,20 +1312,10 @@
 
 template<class T>
   inline constexpr bool @\exposid{is-signed-integer-like}@ = @\seebelow@; @\itcorr[-2]@    // \expos
-
-template<class I>
-  concept @\deflibconcept{weakly_incrementable}@ =
-    @\libconcept{default_initializable}@<I> && @\libconcept{movable}@<I> &&
-    requires(I i) {
-      typename iter_difference_t<I>;
-      requires @\exposid{is-signed-integer-like}@<iter_difference_t<I>>;
-      { ++i } -> @\libconcept{same_as}@<I&>;   // not required to be equality-preserving
-      i++;                      // not required to be equality-preserving
-    };
 \end{codeblock}
```

Comment:
Hm, it's a bit weird to have these "bare" codeblocks, which in reality are stuck to their preceding paragraph. The status quo was sort of OK, but the new codeblock you're now introducing below seems to be stuck to the wrong paragraph now. We're lacking a mechanism to introduce something with an unnumbered block at the beginning. We have that for synopses and for grammar, and I suppose for itemdecls, but this use case is none of these. Thoughts?

Reply:
FWIW, I wouldn't mind if all of these concept definitions were changed to use `itemdecl` instead of `codeblock`. I'm not sure how we landed on `codeblock` in the first place; I suspect it was a typo on my part the first time that was then replicated a couple of dozen times.

Reply:
Rebased. I've also moved the `\pnum` to before the codeblock.

Reply:
I've been adding subclause structures whenever we had more such "bare" codeblocks (of which recent papers added a lot), but maybe itemdecls are the way to go here. In any case, I'm afraid this needs another rebase.

Reply:
@jensmaurer would you like to take another stab at this?